### ISSUE TITLE
SqlServerDsc: Update schema property descriptions with those in README.md

### ISSUE THREAD

**Issue by johlju** [dsccommunity/SqlServerDsc#1568]
To support auto-documentation make sure the property descriptions in the schema is correct. Another PR need to make sure that is correct before removing the parameter documentation from the README.md.

Once that is done the documentation can be removed from the README.md (once the documentation is in the Wiki).

_Originally posted by @johlju in https://github.com/dsccommunity/SqlServerDsc/pull/1567#issuecomment-640734339_

**Comment by malikadilahmed** [comment]
Hello Johan
I see you have contributed tremendously in sql server and I found some of the sql server dsc configurations you have shared, but I never found readme.txt that can explain how to run and execute them. Especially people who are new to PS DSC (like me) have very much difficulty. Could you please add couple of lines regarding how to run your DSC configuraion scripts.
Regards
Mali
